umpost/um2netcdf.py

```diff
@@ -603,6 +604,16 @@ def check_pressure_warnings(need_heaviside_uv, heaviside_uv, need_heaviside_t, h
               "These fields will be skipped")
 
 
+def add_global_history(infile, iris_out):
+    version = -1  # FIXME: determine version
```

Comment:
Once the initial version number is decided, would it be moved to a constant to make it easier to remember to change when there are new updates?

Reply:
I suspect we'll use a versioning utility to auto-handle this (I'll add an issue now).